enterprise/migrations/0132_auto_20210604_1556.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.20 on 2021-06-04 15:56
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('enterprise', '0131_auto_20210517_0924'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='enrollmentnotificationemailtemplate',
+            name='template_type',
+            field=models.CharField(choices=[('DEFAULT_ENROLL', 'Default Enrollment Template'), ('ADMIN_ENROLL', 'Admin Enrollment Template')], default='DEFAULT_ENROLL', help_text='Use either DEFAULT_ENROLL or ADMIN_ENROLL', max_length=255),
+        ),
+        migrations.AddField(
+            model_name='historicalenrollmentnotificationemailtemplate',
+            name='template_type',
+            field=models.CharField(choices=[('DEFAULT_ENROLL', 'Default Enrollment Template'), ('ADMIN_ENROLL', 'Admin Enrollment Template')], default='DEFAULT_ENROLL', help_text='Use either DEFAULT_ENROLL or ADMIN_ENROLL', max_length=255),
+        ),
+        migrations.AlterField(
+            model_name='enrollmentnotificationemailtemplate',
+            name='enterprise_customer',
+            field=models.OneToOneField(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='enterprise_enrollment_template', to='enterprise.EnterpriseCustomer'),
+        ),
+    ]
```

Comment:
@binodpant were you not going to add a migration that populated default templates?

Reply:
Oh yeah I thought "default" in my brain for some reason was filling this in already, but it absolutely does not.
Yeah we need a migration for the bulk enrollment and default template that was file based

Reply:
ah yes that is needed of course! otherwise we will have empty templates..